src/help/faq/README.md

```diff
@@ -25,7 +25,14 @@ If you're having problems with the app, please check here before posting. If you
 ---
 
 ### App crashes on launch
-If you are using the **Public build**, check if you are on iOS 13.4+ or iPadOS 13.4+. If you are using the **Beta / Patron build**, ping @Paper on the Discord #supporter channel.
+If you are using the **Public build**, check if you are on iOS 13.4+ or iPadOS 13.4+. If you are using the **Beta / Patron build**, ping @Paper on the Discord #supporter-chat channel.
```

Comment:
```suggestion
If you are using the **Public build**, ensure that you are on iOS 13.4+ or iPadOS 13.4+. If you are using the **Beta / Patron build**, ping @Paper on the Discord #supporter-chat channel.
```

Reply:
Though note that the Beta/Patreon builds have the exact same versioning requirements as the public.
13.4+ should work on __any__ build. Perhaps you can remove the second line.

Reply:
That should indeed work on any build but I think the idea was
 * if people are on the public build the issue is likely to be caused by iOS < 13.4
 * if people are on the beta build, we assume that they tried the public build before or are informed about the app. thus the issue is likely due to a bug.

We could separate the information:
```suggestion
Ensure that you are on iOS 13.4+ or iPadOS 13.4+. 

If you are using the **Beta / Patron build**, ping @Paper on the Discord #supporter-chat channel.
```
What do you think?

Reply:
Okay, I understand your intentions. If that's the case, I think it's fine to keep them seperate. I have two minor suggestions, then!

* Don't say ping @Paper, there are others in support who may be able to help too! Simply say talk in that discord channel.
* Explain __why__ you should be asking there, instead of describing the issue here. Something along the lines of.. `The Patreon builds are usually unstable. If you are on the correct versioning for your device, and are still getting crashes, you may be encountering a known or unknown bug for your Paperback version. Contact us on our Discord #supporter-chat channel, and staff will try to determine what's happening with you.`

Reply:
You're right, asking to ping Paper do not seems to really make sense.
I'm not sure why it was like that in the beginning but I think we can change that. I will modify the PR using your last sentence.

Reply:
Done in a423f28. I kept the `If you are using the **Beta / Patron build**` sentence to make explicit that not everybody are using the beta build. What do you think?